### PR TITLE
Add ReadFile variable in buildtools/file API.

### PR DESCRIPTION
**Pull request by mohdanishaikh**

### CODE REVIEW

**Comment by vladmos** [comment]
The CI check is failing due to reasons unrelated to this PR, I'll fix them shortly.

File: file/file.go [hunk 1]
```diff
@@ -24,8 +24,10 @@ import (
 	"os"
 )
 
-// ReadFile is like ioutil.ReadFile.
-func ReadFile(name string) ([]byte, os.FileInfo, error) {
+var ReadFile = readFile
```
Comment (vladmos) [inline file/file.go:27]: Could you please add a comment why this is required, otherwise someone can remove it in the future.

**Review by vladmos — APPROVED** [review]

**Comment by vladmos** [comment]
If you rebase to master now the failing tests should disappear.

File: file/file.go (hunk 1 above)
Comment (mohdanishaikh) [reply file/file.go:29]: Thanks. Does the comment look good to you?

**Pull request merged**